docs/docsource/operation/XDM_GUI/bucket_operations/metadata_search.rst

```diff
@@ -0,0 +1,46 @@
+.. _Searching Metadata from |product|:
+
+Searching Metadata from |product|
+=================================
+
+Every object stored in an object storage system is associated with a set
+of metadata. Understanding the metadata tags associated with these objects
```

Comment:
This might be a bit technical but I just want to add it anyway. You describe the system as a "set of metadata". Typically, I see "metadata" described as the entire collection of data and not just a single item. Additionally, set will refer to an unordered unique collection of items ( hashed set is another common term ), see javascripts set https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set. What I think metadata actually is, is a map ( https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map ) or sometimes called a dictionary. I could be wrong here though but from the search options and terms this sounds right to me...

With that said, the best idea I came up with is just removing the "a set of".
```suggestion
Every object stored in an object storage system is associated with metadata. 
Understanding the metadata tags associated with these objects
```

Reply:
Technical is juuust fine...